### DISCUSSION TITLE
这是不是BUG啊？我要咋回到Git岛？？？

### DISCUSSION THREAD

**Discussion by fengfeifang** [ByteLegend/ByteLegend#99] [Q&A]
说出来你都不信，我安装完IDE之后获得了100金币
然后拿着这100金币开开心心的去了Git岛
但是太久没上去了，今天突然上去，就在岛里面了
跟平常看到的画面不一样，我就又坐小海星回了JAVA岛
然后我就没钱回Git岛了
可是我已经完成IDE的任务了
我现在要去哪里搞钱，才能回Git岛呢？？？
……
好像只有好友邀请，可是我又没有邀请码

**Comment by gtn1024** [comment]
你可以尝试我的邀请码：

```
我正在通过游戏学习编程，在爪哇岛的(BJ, 94)坐标处打开
宝箱，输入我的邀请码 uvOHAC2zb3，你和我都可以获得500金币，
快来 https://bytelegend.com 字节传说加入我吧！
```

**Comment by ACodeHX** [comment]
每天登陆都有金币的
